puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
Sure, I'll install the plugin when the instance is quiet (probably over the weekend) and we can check if it'll work next week.  It's probably not worth refactoring this twice.